db/functions/base.py

```diff
@@ -407,6 +407,19 @@ def to_sa_expression(column_expr):
         return mode().within_group(column_expr)
 
 
+class Peak_Time(DBFunction):
```

Comment:
```suggestion
class PeakTime(DBFunction):
```

Please use `CamelCase` for class names

Reply:
Okayy, sorry I missed that.
You can review cascaded PRs later as they have same problem.
I will address them ASAP.

Reply:
Done!